src/Libraries/DSIronPython/DSIronPython.csproj

```diff
@@ -105,9 +115,26 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\class.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\field.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\method.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\namespace.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\privateMethod.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\property.png" />
+  </ItemGroup>
```

Comment:
Any specific reason for moving things from `PythonNodeModelsWpf` to here?

Reply:
These property are being used for the auto-completion feature and we have moved all the auto-completion code to the DSIronPython project. The idea was not to have this PythonNodeModelsWPF project directly depend on any python binaries.